website/source/docs/builders/qemu.html.md.erb

```diff
@@ -116,12 +116,19 @@ Linux server and have not enabled X11 forwarding (`ssh -X`).
     upstream issue which can be tracked
     [here](https://github.com/intel/haxm/issues/20).
 
-    -&gt; The `hvf` accelerator is new and experimental as of
+    -&gt; The `hvf` and `whpx` accelerator are new and experimental as of
     [QEMU 2.12.0](https://wiki.qemu.org/ChangeLog/2.12#Host_support).
-    You may encounter issues unrelated to Packer when using it.  You may need to
+    You may encounter issues unrelated to Packer when using these.  You may need to
     add [ "-global", "virtio-pci.disable-modern=on" ] to `qemuargs` depending on the
     guest operating system.
 
+    -&gt; For `whpx`, note that [Stefan Weil's QEMU for Windows distribution](https://qemu.weilnetz.de/w64/)
+    does not include WHPX support and users may need to compile or source a
+    build of QEMU for Windows themselves with WHPX support. Users can refer
+    to these [instructions (Use on mainline QEMU repository instead)](https://github.com/AlexAltea/orbital/blob/f0fd82b1b7aa0fb18c089b11716731c3b628f504/docs/manual-windows.md)
```

Comment:
I know you're trying to add those extra details to the docs (a link to instructions on how to build qemu with the necessary patch, and a binary with the patch included) in order to be helpful, but I think this kind of information is outside of the scope of Packer's docs. I'd be more comfortable sticking to just the first part, explaining that folks need to build QEMU themselves:

```-&gt; For `whpx`, note that [Stefan Weil's QEMU for Windows distribution](https://qemu.weilnetz.de/w64/)
    does not include WHPX support and users may need to compile or source a
    build of QEMU for Windows themselves with WHPX support.```

And trusting that users who want to use this feature are knowledgable enough about experimental QEMU features to roll their own.

Reply:
OK, that's reasonable. I'll make it so.